packages/pgsearch/node-tests/searcher-test.js

```diff
@@ -170,7 +170,6 @@ describe('pgsearch/searcher', function() {
       attributes: {
         'first-name': 'Quint',
         'last-name': 'Faulkner',
-        'favorite-color': null,
```

Comment:
note: this was an instance of the seed model being modified as part of validation, the original model never had a favorite color.